Add WOPI Business Flow (#789)
owncloud/docs-server: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

* This app requires at minimum ownCloud Version 10.5 and php 7.1.
====

> == Business Flow for Microsoft 365

Reviewer: I would not use the wording "business flow" here as this term in this context relates to the enforced authentication via MS (`wopi.business-flow.enabled`).

Author: Discussed with @tbsbdr - got the point now. Will change the headline.

Author: The headline is fixed now, @tbsbdr pls re-check and approve if ok.

> Apart licensing, when using Microsoft 365, the following business flow applies, contact {oc-support-url}[ownCloud Support] for more details:

Reviewer 2: ```suggestion
Apart from licensing, when using Microsoft 365, the following business flow applies, contact {oc-support-url}[ownCloud Support] for more details:
```

* The customer provides ownCloud:
** a written statement about his Microsoft 365 entitlement.

> ** his ownCloud Instance URL. Only users coming from this URL will be able to use Microsoft 365.

Reviewer 2: ```suggestion
** the URL of their ownCloud instance. Only users coming from this URL will be able to use Microsoft 365.
```

> * ownCloud provides the customer a required proxy URL to be used in the settings, see below.

Reviewer 2: ```suggestion
* ownCloud provides customers with a required proxy URL to be used in the settings, see below.
```

> ** Beside other tasks, the proxy checks if users originate from the given ownCloud Instance URL.

Reviewer 2: ```suggestion
** Among other things, the proxy checks if users originate from the given ownCloud Instance URL.
```

> * When the users opens an Office document via the ownCloud Instance and Office 365 for the web is loaded, users are checked by Microsoft if already signed in with a Microsoft 365 business account. If users are not signed in, they will be prompted to sign in.

Reviewer 2: ```suggestion
* When users open an office document via the ownCloud instance and Office 365 for the web is loaded, Microsoft checks if these users are already signed in via a Microsoft 365 business account. If users are not yet signed in, they will be prompted to sign in.
```

== Preparing the Environment

> To use Microsoft Office for the web, you need an:

Reviewer 2: ```suggestion
To use Microsoft Office for the web, you need:
```

> - {office-online-server-url}[Office Online Server] locally installed, or an 

Reviewer 2: ```suggestion
- an {office-online-server-url}[Office Online Server] locally installed, or 
```

> - active Microsoft 365 subscription including the data provided by ownCloud, see xref:business-flow-for-microsoft-365[Business Flow for Microsoft 365].

Reviewer 2: ```suggestion
- an active Microsoft 365 subscription including the data provided by ownCloud, see xref:business-flow-for-microsoft-365[Business Flow for Microsoft 365].
```
